src/main/java/seedu/contax/ui/onboarding/OnboardingStoryManager.java

```diff
@@ -71,8 +71,8 @@ private void createStories() {
 
 
         test.addStory(new OnboardingStep(
-                "Follow the format 'add n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS'"
-                        + "\n\nExample: add n/Johnny p/91234567 e/Johnny@j.com a/Johnny street"
+                "Follow the format 'addPerson n/NAME p/PHONE_NUMBER e/EMAIL a/ADDRESS'"
```

Comment:
the command is `addperson` no capitalisation

Reply:
woops thanks